src/stylesheets/components/_process-list.scss

```diff
@@ -0,0 +1,62 @@
+// PROCESS LIST
+$bullet-border-width: units($theme-process-list-connection-line-width) / 2;
+$padding-left: ((units($theme-process-list-bullet-size) / 2) - (units($theme-process-list-connection-line-width) / 2));
```

Comment:
Perhaps might want to use a more tightly-scoped variable here, just to prevent any collisions down the road